### PR TITLE
Added Dockerfile.

### PR DESCRIPTION
**Pull request by Manouchehri**

### CODE REVIEW

**Comment by Manouchehri** [comment]
Mind enabling automated builds for the trailofbits org?

image

image

image

image

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:16.04
+MAINTAINER David Manouchehri
+
+RUN apt-get -y update && \
+	DEBIAN_FRONTEND=noninteractive apt-get -y install z3 python-pip git && \
+	git clone https://github.com/trailofbits/manticore.git && \
+	cd manticore && \
+	pip install --no-binary capstone .
```
Comment (japesinator) [inline Dockerfile]: This shouldn't be needed as of #262

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:16.04
+MAINTAINER David Manouchehri
```
Comment (japesinator) [inline Dockerfile]: Would you mind making this one of the core team? (myself, @mossberg, @feliam, @yan)

File: Dockerfile (hunk 1 above)
Comment (Manouchehri) [inline Dockerfile]: Done.

File: Dockerfile (hunk 2 above)
Comment (Manouchehri) [inline Dockerfile]: Done.

**Comment by japesinator** [comment]
Looks good, thanks for the PR!

**Comment by japesinator** [comment]
Automated builds should be enabled at https://hub.docker.com/r/japesinator/manticore/

**Comment by Manouchehri** [comment]
@japesinator Mind triggering the build manually or with `git push`? (If not it'll happen on the next commit anyway.)

**Comment by dguido** [comment]
Hey @Manouchehri, we're going back and collecting CLAs for past contributors of Manticore but we can't seem to find your email. Can you do us a favor and click through this link to sign the Manticore CLA? Thanks! We're happy to accept any more contributions you want to make :-D

https://cla-assistant.io/trailofbits/manticore

**Pull request merged**